### DISCUSSION TITLE
Visual Studio App Center - offline logs

### DISCUSSION THREAD

**Discussion by faizalmrub** [MicrosoftDocs/appcenter-docs#1289] [Q&A]
When there is no connectivity , app center crash / error logs are queued on the local devices. Would we be able to view these logs?

Thanks

**Comment by DmitriyKirakosyan** [comment]
Hi @faizalmrub , logs are stored on a local device while it is offline. All saved logs will be sent to backend once your device gets online and then you will be able to see them on the AppCenter portal.

Also note, that a number of stored logs depends on the local storage size, which is configurable via [Storage size API](https://docs.microsoft.com/en-us/appcenter/sdk/other-apis/ios#storage-size).

**Reply by faizalmrub** [reply]
@DmitriyKirakosyan  is there a way to view the logs on the local device while it is offline? Thanks
